sdk/storage/azure-storage-blob/azure/storage/blob/_shared/models.py

```diff
@@ -487,3 +488,41 @@ def __init__(self):
         self.signed_service = None
         self.signed_version = None
         self.value = None
+
+class TokenAudience():
+    """
+    Base class that defines what calling functions and attributes should be defined for an Azure Active
+    Directory audience token.
+
+    :param str value: The audience to use when requesting tokens for Azure Active Directory authentication.
+        For more information please take a look at:
+        https://learn.microsoft.com/azure/storage/blobs/authorize-access-azure-active-directory. Please use one
+        of the provided classmethods over creating a custom value unless you have a specific scenario for doing so.
+    """
+    DEFAULT_OAUTH_SCOPE = ".default"
+    STORAGE_OAUTH_SCOPE = "https://storage.azure.com/"
+    _value: str
+
+    def __init__(self, value: str) -> None:
+        self._value = value + TokenAudience.DEFAULT_OAUTH_SCOPE
```

Comment:
Did we determine we actually need the `.default`? If we do, we also need to account for if the `value` coming in ends in a `/` or not since customers can also pass any old string. The full scope needs to end in `/.default`.

Reply:
Yeah so when I was testing I ran into some issues. As it turns out `value` is the audience and `.default` is the scope and so you need both (and for the time being we are auto-appending `.default` but there may be configurable scopes coming down the pipeline).

I will also add the logic to handle both `/` and blank endings. I was pattern-matching .NET's test which all ended in `/` but after peeking into the code I see there is logic for handling different endings.